langchain/vectorstores/pinecone.py

```diff
@@ -37,7 +49,9 @@ def __init__(
         index: Any,
         embedding_function: Callable,
         text_key: str,
+        filter: Optional[dict] = None,
```

Comment:
out of curiosity how were you imagining using this? why not just pass in the filter argument to search methods

Reply:
i would just do this by specifying **search_kwargs when converting into a retriever. or are you using for something else besides retrieval?

Reply:
Not really, I can remove it then